src/interface/playerSelection/playerInfo/Gear.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Icon from 'common/Icon';
+import ItemLink from 'common/ItemLink';
+import ITEM_QUALITIES from 'game/ITEM_QUALITIES';
+
+
+class Gear extends React.PureComponent {
+  static propTypes = {
+    gear: PropTypes.array.isRequired,
+  };
+
+  render() {
+    const { gear } = this.props;
+
+    return (
+      <>
+        {
+          gear.filter(item => item.id !== 0)
+          .map(item => {
+            // Items seem to turn epic from 340 item level, but WCL doesn't show this properly
+            let quality = item.itemLevel >= 340 ? ITEM_QUALITIES.EPIC : item.quality;
```

Comment:
Please move 340 to a constant above the class.

Reply:
Done